build.sbt

```diff
@@ -47,30 +48,33 @@ lazy val alpakka = project
       """.stripMargin
   )
 
-lazy val amqp = alpakkaProject("amqp", Dependencies.Amqp)
+lazy val amqp = alpakkaProject("amqp", Dependencies.Amqp, fork in Test := true, parallelExecution in Test := false)
```

Comment:
Something is wrong if adding a new connector implies modifying the way we test all the existing ones :)